### ISSUE TITLE
Accessibility: Voice Over Support

### ISSUE THREAD

**Issue by sportfloh** [zed-industries/zed#7895]
### Check for existing issues

- [X] Completed

### Describe the feature

Zed should be fully accessible via Voice Over.

**Comment by kfjelsted** [comment]
Although some of the menus are accessible I cannot use Voiceover to enter or edit code or text. SUpport of VoiceOver  is a key requirement for accessibility for the Mac!

**Comment by neurodynamic** [comment]
I would also like to see this.
